lib/kamal/commands/builder/native/pack.rb

```diff
@@ -0,0 +1,26 @@
+class Kamal::Commands::Builder::Native::Pack < Kamal::Commands::Builder::Native
+  def push
+    combine \
+      pack(:build,
+        config.absolute_image,
+        "--platform", platform,
+        "--builder", pack_builder,
+        buildpacks,
+        "-t", config.absolute_image,
+        "-t", config.latest_image,
+        "--env", "BP_IMAGE_LABELS=service=#{config.service}",
```

Comment:
Kamal expects there to be a service label, this automatically adds the label via the `paketo-buildpacks/image-labels` buildpack.